app/config/frontendGlobal.scala

```diff
@@ -17,30 +17,28 @@
 package config
 
 import com.typesafe.config.Config
+import javax.inject.Inject
+import models.CGTClientException
 import net.ceedubs.ficus.Ficus._
 import play.api.Play.current
-import play.api.i18n.Messages.Implicits._
-import play.api.mvc.Request
-import play.api.{Application, Configuration, Play}
+import play.api.http.HeaderNames.CACHE_CONTROL
+import play.api.http.Status.{BAD_REQUEST, NOT_FOUND}
+import play.api.i18n.MessagesApi
+import play.api.mvc.Results.{BadRequest, NotFound}
+import play.api.mvc.{Request, RequestHeader, Result}
+import play.api.{Configuration, DefaultGlobal, Logger, Play}
 import play.twirl.api.Html
-import uk.gov.hmrc.play.config.{AppName, ControllerConfig}
-import uk.gov.hmrc.play.frontend.bootstrap.DefaultFrontendGlobal
-import uk.gov.hmrc.play.frontend.filters.{FrontendAuditFilter, FrontendLoggingFilter, MicroserviceFilterSupport}
+import uk.gov.hmrc.play.bootstrap.http.{ApplicationException, FrontendErrorHandler}
+import uk.gov.hmrc.play.config.ControllerConfig
 
-object FrontendGlobal
-  extends DefaultFrontendGlobal {
+import scala.concurrent.Future
+import scala.util.control.NonFatal
```

Comment:
Remove this, the onError and logError functions.  Just replace with DefaultGlobal.onError(request, e)?